haas/rest.py

```diff
@@ -110,7 +110,9 @@ def rest_call(method, path, schema):
 
     * path - the url-path to map the function to. The format is the same as for
             flask's router (e.g. '/foo/<bar>/baz')
-    * method - the HTTP method for the api call (e.g. POST, GET...)
+    * method - the HTTP method for the api call (e.g. POST, GET...).
+               Can also pass a list of methods to support multiple (used in
+               testing)
```

Comment:
The list option is indeed to support testing, but others might find it useful. I think it saves us a bunch of space while keeping the code very understandable and adhering to DRY.

When I tried the stacking option I got an error:
Code:
```python
    @rest.rest_call('GET', '/mixed/args/<arg1>', Schema({
        'arg1': basestring,
        'arg2': basestring,
    }))
    @rest.rest_call('PUT', '/mixed/args/<arg1>', Schema({
        'arg1': basestring,
        'arg2': basestring,
    }))
    def mixed_args(arg1, arg2):
        return json.dumps([arg1, arg2])
```
Error:
```python
            if old_func is not None and old_func != view_func:
                raise AssertionError('View function mapping is overwriting an '
>                                    'existing endpoint function: %s' % endpoint)
E               AssertionError: View function mapping is overwriting an existing endpoint function: mixed_args

venv/local/lib/python2.7/site-packages/flask/app.py:984: AssertionError
```